src/components/DraftImportCsvModal.vue

```diff
@@ -59,43 +54,44 @@ import {
   IonToolbar,
   modalController
 } from "@ionic/vue";
-import { defineComponent } from "vue";
+import { defineProps, ref } from "vue";
 import { closeOutline, saveOutline } from "ionicons/icons";
 import { translate } from '@/i18n'
+import { showToast } from "@/utils";
+
+
+const props = defineProps(["fileColumns", "content", "countId"])
+const productIdentifications = process.env.VUE_APP_PRDT_IDENT ? JSON.parse(process.env.VUE_APP_PRDT_IDENT) : []
+
+const selectedIdentifier = ref('')
+const selectedColumn = ref('')
+
+function closeModal(identifierData: any) {
+  if (!identifierData || !identifierData.length) {
+    identifierData = null;
+  }
+  modalController.dismiss({ dismissed: true, identifierData });
+}
 
-export default defineComponent({
-  name: "DraftImportCsvModal",
-  components: { 
-    IonButton,
-    IonButtons,
-    IonContent,
-    IonFab,
-    IonFabButton,
-    IonHeader,
-    IonIcon,
-    IonItem,
-    IonList,
-    IonListHeader,
-    IonRadioGroup,
-    IonRadio,
-    IonSelect,
-    IonSelectOption,
-    IonTitle,
-    IonToolbar,
-  },
-  methods: {
-    closeModal() {
-      modalController.dismiss({ dismissed: true});
-    },
-  },
-  setup() {
+function saveImportData() {
+  if (!selectedIdentifier.value) {
+    showToast(translate("Please select a Product identifier"));
+  }
 
+  if (!selectedColumn.value) {
+    showToast(translate("Please select the column that corresponds to the product identifier"));
+  }
+
+  const identifierData = props.content.map((row: any) => {
     return {
-      closeOutline,
-      saveOutline,
-      translate
-    };
-  },
-});
+      idType: selectedIdentifier.value,
+      idValue: row[selectedColumn.value]
+    }
+  })
```

Comment:
Currently we are passing an array of objects [{ idType, idValue }] for each entry for csv. But as we will be having the same idType for all the csv entries, we can send is separately and pass the idValue as an array.

Something like
```js
identifierData = {idType: Array of idValues}
```